microros_esp32_extensions/sdkconfig.defaults

```diff
@@ -1,3 +1,4 @@
 CONFIG_ESP_MAIN_TASK_STACK_SIZE=3000
 CONFIG_FREERTOS_UNICORE=y
 CONFIG_ESP_TASK_WDT=n
+CONFIG_RTCIO_SUPPORT_RTC_GPIO_DESC=y
```

Comment:
Is there a possibility that we can set this as an application-specific configuration?

Reply:
Related to: https://github.com/micro-ROS/micro_ros_setup/pull/176#discussion_r514341827

Maybe we can just explain how to configure this in the app README.md.

Reply:
I'll write a detailed REAMDE.md and add it to app.